en-US/S.xml

```diff
@@ -346,6 +360,15 @@
 			</listitem>
 
 		</varlistentry>
+		 <varlistentry id="sign-on">
+			<term>sign-on, sign on</term>
```

Comment:
Similar to the entry for "log in", I suggest adding an entry with the preferred preposition  - I'm assuming "sign on to".